crates/bin/pclientd/src/proxy.rs

```diff
@@ -1,40 +1,28 @@
 use futures::FutureExt;
-use http_body::Body as _;
 use std::convert::Infallible;
 use std::pin::Pin;
 use std::{
     future::Future,
     task::{Context, Poll},
 };
-use tonic::transport::NamedService;
-use tonic::{
-    body::BoxBody,
-    transport::{Body, Channel},
-};
+use tonic::server::NamedService;
+use tonic::{body::BoxBody, transport::Channel};
 use tower::ServiceExt;
 
 fn proxy(
     channel: Channel,
-    req: http::Request<Body>,
+    req: http::Request<BoxBody>,
 ) -> Pin<Box<dyn Future<Output = Result<http::Response<BoxBody>, Infallible>> + Send + 'static>> {
     tracing::debug!(headers = ?req.headers(), "proxying request");
-    // Convert request types
-    let req = req.map(|b| {
```

Comment:
This conversion seems no longer necessary because `BoxBody` is an aliaes for `UnsyncBoxBody<Bytes, Status>`